test/unit/main_test.ts

```diff
@@ -79,7 +79,7 @@ describe('Recognizes invalid TypeScript inputs', () => {
 describe('Visitor pattern', () => {
   it('renames identifiers of property declarations and property access expressions', () => {
```

Comment:
change description. Probably will be too long so you can split to multiple tests.